scripts/mkcloud

```diff
@@ -849,12 +849,12 @@ function start_vbmc
     local ipmi_user=$3
     local ipmi_password=$4
     # make sure old vbmc is not interfering
-    vbmc stop $ironic_node || true
-    vbmc delete $ironic_node || true
+    $sudo vbmc stop $ironic_node || true
```

Comment:
I'm looking e.g. in https://opendev.org/openstack/ironic/src/branch/master/devstack/lib/ironic#L1818-L1828 and it doesn't seem like sudo should be needed to use vbmc?

Reply:
I haven't used devstack in a while but maybe there's a difference in how the libvirt auth is configured and/or instances are not started as root. In our jobs I get this:
```
13:12:52 libvirt: Polkit error : authentication unavailable: no polkit agent available to authenticate action 'org.libvirt.unix.monitor'
13:12:52 2019-07-24 13:12:52,004.004 31391 ERROR VirtualBMC [-] Command failed: Fail to establish a connection with libvirt URI "qemu:///system". Error: authentication unavailable: no polkit agent available to authenticate action 'org.libvirt.unix.monitor'

Reply:
You can add the user to the libvirt group to allow them to have non-root access to the libvirt interface, maybe devstack is set up that way. Probably not worth the hassle here.